Parameter devel2 (#51)
campa-consortium/pals: source/element-parameters.md

```

%---------------------------------------------------------------------------------------------------
## Dependent and User Set parameters.

> Some parameters are set in the lattice file. These parameters are called "User settable". 

Reviewer: I'm not sure about the word "settable".  I suggest calling these "User set" parameters, using the same phrase that appears in the title of this section.

Author: The problem with "User set" is that this implies that the User has set the parameter. As opposed to "User settable" which implies that the User **may** set the parameter.

> Some parameters are not settable and will be computed by lattice expansion code. 

Reviewer: ```suggestion
Some parameters cannot be set by the user and will be computed by lattice expansion code. 
```

Author: changed to:
```
Some parameters will be computed by the Translator during lattice expansion. Such parameters
may not be set by the user.
```

Author: This brings up the question whether it is an error for Translator computed quantities to be present in the lattice file. I can see a use case where the User wants to print out the Translated file complete with all computed quantities.

For example, the `FloorP` parameters can be set for the `BeginningEle` element which is the
first element of any branch line. For most other elements, the `FloorP` parameters can

> be calculated starting at the `BeginningEle` and working 

Reviewer: This section ends abruptly.  Maybe it would be good to add "(... under construction)" or something similar.

Author: Added "and working forward computing the floor parameters
element-by-element."
